squape/squishserver.py

```diff
@@ -0,0 +1,52 @@
+import squish
+import test
+from remotesystem import RemoteSystem
+import os
+import pathlib
+from squape.report import log
+from squape.internal.exceptions import SquishserverError
+
+class SquishServer:
+    def __init__(self, location=None, host='127.0.0.1', port=4322):
+        if location is None:
+            self.location = os.environ['SQUISH_PREFIX']
+
+        self.host = host
+        self.port = port
+        self.remotesys = RemoteSystem(host, port)
```

Comment:
Please handle an exception when connection to squishserver cannot be established. https://doc.qt.io/squish/squish-api.html#remotesystem-remotesystem